src/NuGet.config

```diff
@@ -4,7 +4,7 @@
     <clear />
     <add key="nuget.org" value="https://www.nuget.org/api/v2/" />
     <add key="MyGetApplicationInsightsSdkLabs" value="https://www.myget.org/F/applicationinsights-sdk-labs" />
-    <!--<add key="Library.NuGet release" value="D:\Git\ApplicationInsights-LocalForwarder\src\Library.NuGet\bin\Release" />-->
```

Comment:
Is myget  intended? It's fine though

Reply:
Yes, the SDK RC is there right now, I'll change it to nuget.org once it's officially out.